sdk/cosmosdb/Azure.ResourceManager.CosmosDB/src/autorest.md

```diff
@@ -133,7 +133,6 @@ rename-mapping:
   CassandraTablePropertiesResource: ExtendedCassandraTableResourceInfo
   CassandraTableResource: CassandraTableResourceInfo
   CosmosTablePropertiesResource: ExtendedCosmosTableResourceInfo
-  DatabaseRestoreResource: DatabaseRestoreResourceInfo
```

Comment:
Please keep this line as `Resource` has a special meaning in .NET SDK, model name cannot end with Resource.

Reply:
Reverted back